components/dashboards-web-component/src/designer/components/WidgetConfigurationPanel.jsx

```diff
@@ -106,15 +202,158 @@ class WidgetConfigurationPanel extends React.Component {
         return publishers;
     }
 
+    getPreferences() {
```

Comment:
Lets use a more appropriate name for this function. This does NOT return "preferences", rather it returns input elements for the options of the current selected widget. So how about `getCurrentWidgetOptionsInputs`.

Reply:
changed with 910980c413d9aea940c47a751f949593c1fc5ec0